src/containers/reCaptcha/utils.ts

```diff
@@ -0,0 +1,32 @@
+"use client";
+
+import { useEffect, useLayoutEffect } from "react";
+
+/**
+ * Function to generate the src for the script tag
+ * Refs: https://developers.google.com/recaptcha/docs/loading
+ */
+export const getRecaptchaScriptSrc = ({
```

Comment:
This seems like something that could live in the recaptcah service. Don't thing we need another utils.
If it's a generic util that we want to expose to the rest of the app, then it should live in the general utils dir.